### ISSUE TITLE
#### STARTUP ####

### ISSUE THREAD

**Issue by Tabbleman** [LuaLS/vscode-lua#99]
### Which OS are you using?

Linux

#### STARTUP ####
[2023-03-06 22:39:35] |   DEBUG   |     Filesystem     | Created directory at "/home/z/.config/Code/User/globalStorage/sumneko.lua/addonManager"
[2023-03-06 22:39:35] |   ERROR   |         Git        | Failed to checkout repo!
[2023-03-06 22:39:35] |   ERROR   |         Git        | error: pathspec 'main' did not match any file(s) known to git.

### Actual Behaviour

...

I also see the following error:

```
[2023-03-06 22:39:35] |   ERROR   |         Git        | error: pathspec 'main' did not match any file(s) known to git.

```

### Reproduction steps

2. Click '...'
3. See error '...'

**Comment by carsakiller** [comment]
I believe this has been fixed by 5a23814108a594b15c44fb6e961ba7a0c15ce70c. If you can confirm by installing one of the `vsix` files from the [latest action run](https://github.com/LuaLS/vscode-lua/actions/runs/4337871587), it would be greatly appreciated 🙂. Otherwise, there should be a release soon that will contain the fix.

**Comment by Tabbleman** [comment]
thx for your reply, problem fixed.@carsakiller
